docs/pages/bare/using-expo-client.md

```diff
@@ -1,49 +1,69 @@
 ---
-title: Using Expo Go in Bare Workflow
+title: Custom native code in Expo Go
 sidebar_title: Using Expo Go
 ---
 
-import ImageSpotlight from '~/components/plugins/ImageSpotlight'
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { InlineCode } from '~/components/base/code';
+import SnackInline from '~/components/plugins/SnackInline'
 
-It's not currently possible to include your own native code in Expo Go, so it might surprise you to learn that it is still possible to run your bare project in the Expo Go app.
+The [Expo Go][expo-go] app is capable of running most React Native apps regardless of how they were bootstrapped. Developing any React Native app in Expo Go is useful because you can distribute your project instantly with the members of your [organization](/accounts/account-types/#organizations), no native build or provisioning required.
```

Comment:
```suggestion
The [Expo Go][expo-go] app is capable of running most React Native apps regardless of how they are bootstrapped. Developing any React Native app in Expo Go is useful because you can distribute your project instantly to the members of your [organization](/accounts/account-types/#organizations), and no native build or provisioning is required.
```